projects/plugins/jetpack/changelog/class.jetpack-post-images.php

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: bugfix
+
+Post Images: Ensure $post is a WP_Post instance.
```

Comment:
I would recommend being a bit more generic in here, since that changelog entry will be displayed to all end-users when they update the Jetpack plugin. The entry consequently has to be as user-friendly as possible. How about something like:

```suggestion
Post Images: avoid PHP notices when fetching images from posts with missing metadata.
```

Reply:
Yes, this makes much more sense. Thanks for the suggestion!